theme/js/components/utils/featured.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+<div class="fr-toggle fr-toggle--label-left">
+    <input @click="toggleFeatured" type="checkbox" class="fr-toggle__input" aria-describedby="toggle-698-hint-text" id="toggle-698" :checked=featured>
```

Comment:
We don't have this `id` or another hint text so I think we can remove this one. Also, I added missing quotes. 
```suggestion
    <input @click="toggleFeatured" type="checkbox" class="fr-toggle__input" id="featured-toggle" :checked="featured">
```